bazel-java-deps.bzl

```diff
@@ -50,7 +69,7 @@ def install_java_deps():
             "com.github.scopt:scopt_{}:4.0.0".format(scala_major_version),
             "com.google.code.findbugs:jsr305:3.0.2",
             "com.google.code.gson:gson:2.8.2",
-            "com.google.guava:guava:29.0-jre",
+            "com.google.guava:guava:{}".format(guava_version),
```

Comment:
given that we only seem to use that once, I’m not sure factoring it out buys us anything.

Reply:
i just did it because `guava` is one of the version-sensitive libraries in this grpc+netty+protobuf+scalapb bundle - so it was more to bring the version front and center to be updated with the other rather than refactoring to avoid repetition